static/js/d3_histogram_plot.html

```diff
@@ -355,38 +355,38 @@
 }
 
 function getUrlVars() {
-    var vars = {};
```

Comment:
tabs were set as initial indentation in parent file, but I agree difference in indentation it too annoying, and I prefer to use spaces in all new files in our project